app/src/main/res/layout/fragment_home.xml

```diff
@@ -1,50 +1,51 @@
-<?xml version="1.0" encoding="utf-8"?>
-<!-- This Source Code Form is subject to the terms of the Mozilla Public
+<?xml version="1.0" encoding="utf-8"?><!-- This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Did you change this file at all? It actually looks like ALL you did was add the `clipChildren=false` in the RecyclerView at the end. Can you revert this whole file and then just include that one change?

`git rebase -i` this commit, and then `edit` it
`git checkout app/src/.../fragment_home.xml`